tests/acceptance/features/apiGraph/roleManagementEndpoint.feature

```diff
@@ -8,6 +8,7 @@ Feature: permissions role definitions
 
 
   Scenario: get a list of permissions role definitions
+    Given the administrator has enabled the permissions role "Secure Viewer"
```

Comment:
maybe we should add separate scenario with secure viewer enabled and leave it as the test for default state